test/topology/test_coordinator_queue_management.py

```diff
@@ -68,10 +68,7 @@ async def test_coordinator_queue_management(manager: ManagerClient):
 
     await wait_for_first_completed([l.wait_for("received request to join from host_id", m) for l, m in zip(logs[:3], marks[:3])])
 
-    # FIXME: we aren't actually awaiting this log -- this line is missing an `await`.
-    # But this log was actually removed in commit d576ed31dce292997d1cf32af5a9e89768b154d7.
-    # Should we be waiting for something else, or for nothing at all?
```

Comment:
@gleb-cloudius please check this.

I think the log message was removed by accident in d576ed31dce292997d1cf32af5a9e89768b154d7.

Reply:
Yes, and because await was missing we did not catch it.